collections/nemo_nlp/nemo_nlp/modules/losses.py

```diff
@@ -120,12 +120,16 @@ def __init__(self, num_classes, **kwargs):
         self._criterion = nn.CrossEntropyLoss()
         self.num_classes = num_classes
 
-    def _loss_function(self, logits, labels, input_mask):
-        active_loss = input_mask.view(-1) > 0.5
+    def _loss_function(self, logits, labels, loss_mask):
+        active_loss = loss_mask.view(-1)
```

Comment:
why did we remove "> 0.5"

Reply:
because we're now using loss_mask which contains True/False not 1/0 as the input_mask

Reply:
that makes sense. we'd update the docstring to explain the data type each input takes.

Reply:
updated the docstring